### PR TITLE
Add stub data and skeleton components

### PR DESCRIPTION
**Pull request by gnilekaw**
These are the results of "mobbing" with @anandaroop and @oxaudo. We have some new fake data and beautiful orange and pink divs. One thing Roop and I discussed was how to pass messages from the sidebar to the content pane. In the past I've used React Router for this. Roop mentioned Waypoints might possibly be relevant also.

screen shot 2017-04-04 at 3 14 45 pm

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please assign someone to your PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by oxaudo** [comment]
Thanks @anandaroop and @gnilekaw for letting me crash your pairing session. Excited to get into this more:)

I think we'll need to make the tests great again though before we could merge it:(

File: desktop/apps/categories2/components/GeneFamilies.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
```
Comment (oxaudo) [inline desktop/apps/categories2/components/GeneFamilies.js]: Is this react convention to keep the things that are supposed to come as a list in one file? Like here GeneFamilies and GeneFamily come together?

Comment (gnilekaw) [inline desktop/apps/categories2/components/GeneFamilies.js]: Not really. It's best to keep components in their own files. So `GeneFamily` should be in its own component. But for now, while we're sketching this out, I think it's fine.

**Comment by gnilekaw** [comment]
@oxaudo Do you want to put this in columns in this PR?

File: desktop/apps/categories2/components/Sidebar.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+class Sidebar extends React.Component {
+  constructor(props){
+    super(props)
+  }
+
+  items(){
+    const items = this.props.data
+    return items.map((value, index) => {
+      return <li key={`gene-${index}`}>{value.name}</li>
```
Comment (anandaroop) [inline desktop/apps/categories2/components/Sidebar.js]: The fixture data includes the BSON IDs, so maybe we could just start using `value.id` already as the `key`?

File: desktop/apps/categories2/components/__snapshots__/AppContainer.spec.js.snap [hunk 3]
```diff
@@ -1,7 +1,109 @@
...
 
 exports[`renders correctly 1`] = `
-<h1>
-  The Art Genome Project
-</h1>
+<div>
+  <div
+    style={
+      Object {
+        "backgroundColor": "orange",
```
Comment (anandaroop) [inline desktop/apps/categories2/components/__snapshots__/AppContainer.spec.js.snap]: Hmm, this is a funny looking representation of the inline styles — I wonder if this is what Jest snapshots always look like for these?

File: desktop/apps/categories2/components/AppContainer.js [hunk 4]
```diff
@@ -1,8 +1,45 @@
...
 import React from 'react'
+import Intro from './Intro'
+import Sidebar from './Sidebar'
+import Content from './Content'
+import GeneFamilies from './GeneFamilies'
 
 class AppContainer extends React.Component {
+  constructor(props){
+    super(props)
+    this.contentStyles = this.contentStyles.bind(this)
```
Comment (gnilekaw) [inline desktop/apps/categories2/components/AppContainer.js]: You're right. Totally not necessary until those functions start wanting to use props and state.

**Comment by oxaudo** [comment]
Sorry guys. 

I accidentally messed up and merged those changes when I was trying to push my changes to this branch. Instead I've merged it with my changes directly to master:(

Granted this is an unused application and all of it just prototype at this point - I think it's easier not to revert it and just move forward. 

My changes included the style changes - where I actually moved away from the component styles and just defined styles as regular css classes that got included through `index.styl` in stylesheets folder of categories2 app.

Currently looks like this:
screen shot 2017-04-05 at 10 28 04 am

I believe @gnilekaw will follow up with a pr that will include changes to address @anandaroop comments in the pr here.

**Comment by anandaroop** [comment]
Thanks for "accidentally" improving the styles @oxaudo ! 

Btw, aside from this being an admin-only page, the merge itself was into the `ps-categories2` feature branch, not `master` so no harm done really.

The only thing I'd note is that if we stick with an external css stylesheet for this app, we _may_ want to  [BEM](https://en.bem.info/methodology/quick-start/)-ify the class names to be safer/more consistent with Force. Something to keep in mind.

**Pull request merged**